README.md

```diff
@@ -27,6 +27,14 @@ creating notes and occurrences in Grafeas. There are client libraries available
 * The authoritative API for grafeas is the [protobuf
 files](https://github.com/Grafeas/Grafeas/tree/master/proto/v1beta1).
 
+## Storage
+
+Grafeas could support different storages:
```

Comment:
nit: let's rephrase this as "The following projects provide bindings for Grafeas API to different storage backends:"

Reply:
Done!